test/clientconn_state_transition_test.go

```diff
@@ -206,6 +230,8 @@ func testStateTransitionSingleAddress(t *testing.T, want []connectivity.State, s
 }
 
 // When a READY connection is closed, the client enters IDLE then CONNECTING.
```

Comment:
nit: This comment isn't associated to any test now. I think we should remove it or combine it with the new comment.

Reply:
Oops. Removed.